src/main/java/core/basesyntax/imp/FruitMinus.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.imp;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecordDto;
+import core.basesyntax.model.Storage;
+import core.basesyntax.service.FruitOperation;
+
+public class FruitMinus implements FruitOperation {
+    @Override
+    public boolean operation(FruitRecordDto fruitRecordDto) {
+        if (fruitRecordDto == null) {
+            return false;
+        }
+        ValidatorImp validator = new ValidatorImp();
```

Comment:
```suggestion
        Validator validator = new ValidatorImp();
```
reference should be more abstract (interface)